data_pipelines_cli/cli_commands/create.py

```diff
@@ -54,7 +54,7 @@ def create(project_path: str, template_path: Optional[str], vcs_ref: str) -> Non
     config = read_env_config()
     config_templates = config["templates"]
     src_template_path = _get_template_path(config_templates, template_path)
-    copier.copy(src_path=src_template_path, dst_path=project_path, vcs_ref=vcs_ref)
+    copier.run_auto(src_path=src_template_path, dst_path=project_path, vcs_ref=vcs_ref)
```

Comment:
What is the difference?

Reply:
In previous releases, they were using `copy` and this changed to `run_auto` obviously they are still giving us backward compatibility but I was thinking of upgrading this as well.

example from their codebase
```
from .main import run_auto as copy  # noqa: F401; Backwards compatibility
```